crd/src/util.rs

```diff
@@ -71,33 +95,63 @@ fn check_chroot(chroot: Option<&str>) -> ZookeeperOperatorResult<Option<&str>> {
         });
     }
 
-    let reserved_words = vec![".", "..", "zookeeper"];
+    let _ = is_valid_node(chroot)?;
```

Comment:
Do you need to assign?
Wouldn't this work as well

```suggestion
    is_valid_node(chroot)?;
```